client/src/Components/Footer/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import {
+  Row, Col, Image, Typography,
+  Button, Space, Tooltip, Layout,
+} from 'antd';
+import {
+  PhoneOutlined, MailOutlined, TwitterOutlined,
+  createFromIconfontCN, InstagramOutlined, WhatsAppOutlined,
+  YoutubeOutlined,
+} from '@ant-design/icons';
+import logo from '../../assets/images/logo.svg';
+import map from '../../assets/images/map.svg';
+import './footer.css';
+
+const { Footer } = Layout;
+
+const IconFont = createFromIconfontCN({
+  scriptUrl: [
+    '//at.alicdn.com/t/font_1788044_0dwu4guekcwr.js', // icon-javascript, icon-java, icon-shoppingcart (overrided)
+    '//at.alicdn.com/t/font_1788592_a5xf2bdic3u.js', // icon-shoppingcart, icon-python
+  ],
+});
+
+const { Title, Paragraph, Text } = Typography;
+
+function MainFooter() {
+  return (
+    <Row
+      justify="space-between"
+      align="top"
+      gutter={[10, 30]}
+      className="footer-row"
+    >
+      <Col
+        xs={{ span: 23 }}
+        sm={{ span: 12 }}
+        md={{ span: 12 }}
+        lg={{ span: 7 }}
+        xl={{ span: 7 }}
+      >
+        <Image src={logo} />
+        <Paragraph type="secondary" strong>
+          هذا النص هو مثال لنص يمكن أن يستبدل في نفس المساحة،
+          لقد تم توليد هذا النص من مولد النص العربى،
+          حيث يمكنك أن تولد مثل هذا النص.
+
+        </Paragraph>
+      </Col>
+      <Col
+        xs={{ span: 23 }}
+        sm={{ span: 12 }}
+        md={{ span: 12 }}
+        lg={{ span: 4 }}
+        xl={{ span: 4 }}
+      >
+        <Title level={4}>خدماتنا</Title>
+        <Space direction="vertical">
+          <Text strong type="secondary">تنظيف المنازل</Text>
+          <Text strong type="secondary">تنظيف السجاد</Text>
+          <Text strong type="secondary">تنظيف النوافذ</Text>
+          <Text strong type="secondary">تنظيف السيارات</Text>
+          <Text strong type="secondary">التنظيف التجاري</Text>
+          <Text strong type="secondary">تنظيف بعد البناء</Text>
+        </Space>
+      </Col>
+      <Col
+        xs={{ span: 23 }}
+        sm={{ span: 12 }}
+        md={{ span: 12 }}
+        lg={{ span: 4 }}
+        xl={{ span: 4 }}
+      >
+        <Title level={4}>تواصل معنا</Title>
+        <Space direction="vertical">
+          <Text strong type="secondary">
+            <PhoneOutlined style={{ color: '#00ADEE', marginLeft: '5px' }} />
+            0599000000
```

Comment:
There should be a json file for the site configs and details such like the phone number and the email